onnxruntime/core/graph/graph_utils.cc

```diff
@@ -383,6 +383,10 @@ bool AllNodeInputsAreConstant(const Graph& graph, const Node& node) {
   return true;
 }
 
+bool ContainsSubgraph(Node& node) {
+  return !node.GetAttributeNameToMutableSubgraphMap().empty();
```

Comment:
Ideally this would work on a const node. Could implement this in a slightly less efficient way (see ForAllSubgraphs for example of iterating the attributes to see if any return true for has_g()), or could add ContainsSubgraph to the Node class.